gitlab/Caddyfile-socket

```diff
@@ -0,0 +1,24 @@
+https://dev.sth.se.k0nsl.org {
+
+    log git.access.log {
+        rotate_size 50 # Rotate after 50 MB
+        rotate_age  80 # Keep rotated files for 90 days	
+        rotate_keep 15 # Keep at most 20 log files
```

Comment:
You can drop these rotate directives, they're not relevant to the example; in fact, maybe the whole log can go.